x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -468,6 +468,12 @@ public class Security extends Plugin
         License.OperationMode.ENTERPRISE
     );
 
+    public static final LicensedFeature.Momentary USER_PROFILE_COLLABORATION_FEATURE = LicensedFeature.momentary(
+        null,
+        "user-profile-collaboration",
+        License.OperationMode.PLATINUM
```

Comment:
Should this be STANDARD instead?
Given the final conclusion in https://github.com/elastic/dev/issues/2109#issuecomment-1242160178, we want this feature to be available in STANDARD in Cloud, that means PLATINUM in self-managed (since STANDARD and GOLD are not available anymore)

Reply:
You are right. I got confused about how Cloud tracks this. I have updated it to `STANDARD`.